src/newcharacter.cpp

```diff
@@ -1063,11 +1063,14 @@ void set_points( tab_manager &tabs, avatar &u, pool_type &pool )
     } while( true );
 }
 
+static const std::array<std::string, 4> stat_labels = { _( "Strength" ), _( "Dexterity" ), _( "Intelligence" ), _( "Perception" ) };
```

Comment:
I think all you need to do here is move this inside set_stats(), then it will be a function level static with deferred initialization.

Reply:
I intend to follow this up with consolidating all the places that reproduce this list. I've pushed a change that I think will work here, and indicating how I'd proceed on a larger scale if this is acceptable and works.

Reply:
Please let me know if the update is also unacceptable. If so then I'll move it to a normal function level static and make the global a separate PR when I deduplicate all the uses.